### ISSUE TITLE
[SF][A11y][0.26.0][MessageBox]modal dialog announcement of MessageBox is missing

### ISSUE THREAD

**Issue by ksblklu** [SAP/ui5-webcomponents-react#3208]
**Describe the bug**
Modal dialog announcement is missing after opening MessageBox Component

**To Reproduce**
1. Go to [MessageBox Document Page](https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-messagebox--default-story)
2. Click on 'open MessageBox' button
3. Inspect speech history of JAWS screen reader
4. See announcement of MessageBox modal dialog is missing

<img width="1707" alt="image" src="https://user-images.githubusercontent.com/27538066/177506399-eb1ff6d9-5c45-4c51-b124-7069dc1cd520.png">

**Expected behavior**
Modal dialog message should be announced after the MessageBox opens, same as dialog:
<img width="1745" alt="image" src="https://user-images.githubusercontent.com/27538066/177506802-07b2f279-8266-4329-a896-c3aa858c8fee.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.5.0
`@ui5/webcomponents-react` version: 0.26.0
Operating System: Windows
Browser: Chrome
